### PR TITLE
[PRTL-2887] add number check to print p-value

### PR DESCRIPTION
**Pull request by samrichca**
## Ticket Number

e.g. PRTL-0000

## Environment to be used in testing

- [x] `prod`
- [x] `dev-oicr`
- [x] `qa-yellow`

## Description of Changes

- add default for `buttonStyle` to fix a page crash
- change `pValue` check - look for a number, not just a truthy value
- refactor - import specific lodash functions

### CODE REVIEW

File: src/packages/@ncigdc/components/SurvivalPlotWrapper.js [hunk 1]
```diff
@@ -211,7 +211,6 @@ const SurvivalPlotWrapper = ({
                 ))}
             </Row>
           </div>
-
```
Comment (caravinci) [inline src/packages/@ncigdc/components/SurvivalPlotWrapper.js]: Why is this change necessary?

I was trying to fix it using github, but it messes the spacing.
Mind undoing it?

Comment (samrichca) [reply src/packages/@ncigdc/components/SurvivalPlotWrapper.js]: merge funkiness probably. hang on
